extras/gizmo/transform-gizmo.js

```diff
@@ -430,10 +439,36 @@ class TransformGizmo extends Gizmo {
         return this._meshColors.axis.z;
     }
 
+    /**
+     * The color alpha for all axes.
+     *
+     * @type {number}
+     */
+    set colorAlpha(value) {
+        this._colorAlpha = math.clamp(value, 0, 1);
+
+        this._updateAxisColor('x', this._meshColors.axis.x);
+        this._updateAxisColor('y', this._meshColors.axis.y);
+        this._updateAxisColor('z', this._meshColors.axis.z);
```

Comment:
missing xyz & face?

Reply:
This should do all of them but you will need to set it for each of the Translate Rotate and Scale gizmo class instances

Reply:
Oh you mean the center igy

Reply:
Okay added 👍